raft/raft_test.go

```diff
@@ -1545,7 +1545,7 @@ func testRecvMsgVote(t *testing.T, msgType pb.MessageType) {
 
 		msgs := sm.readMessages()
 		if g := len(msgs); g != 1 {
-			t.Fatalf("#%d: len(msgs) = %d, want 1", i, g)
+			t.Errorf("#%d: len(msgs) = %d, want 1", i, g)
 			continue
```

Comment:
cc @ahrtr for advice.

Reply:
I think it makes more sense just to remove the `continue`, obviously it isn't needed because it will never be executed.

Reply:
> I think it makes more sense just to remove the `continue`, obviously it isn't needed because it will never be executed.

Update the code as suggested.